### PR TITLE
Transaction Logging Python Skeleton

### PR DESCRIPTION
**Pull request by zclarkEDC**
Skeleton of Transaction Logging Microservice in Python Chalice. 
Set up partial DB integration for creating and retrieving simple items.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.chalice/deployments/
+.chalice/venv/
```
Comment (folksgl) [inline .gitignore:2]: Please add .chalice/deployed/

File: app/.chalice/config.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "version": "2.0",
+  "app_name": "app",
```
Comment (folksgl) [inline app/.chalice/config.json:3]: The app_name should match the pattern:
github repo = identity-give-<app_name>
so here it should be "transaction-logging"

File: app/.chalice/deployed/dev.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (folksgl) [inline app/.chalice/deployed/dev.json:1]: This needs to be taken out of the git history. We're not tracking chalice state files.

**Review by folksgl — CHANGES_REQUESTED** [review]

File: app/.chalice/deployed/dev.json (hunk 3 above)
Comment (zclarkEDC) [reply app/.chalice/deployed/dev.json]: fixed with [removed deployed folder](https://github.com/18F/identity-give-transaction-logging/pull/5/commits/84a4678c14923577777dda038ede549c1be9861a)

**Pull request closed without merging**